### PR TITLE
Add support for PR reviews in Github processor

### PR DESCRIPTION
**Pull request by sayanchowdhury**
Signed-off-by: Sayan Chowdhury <sayan.chowdhury2012@gmail.com>

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/tests/github.py [hunk 1]
```diff
@@ -661,143 +664,1198 @@ class TestGithubIssueComment(Base):
...
                 "html_url": "https://github.com/fedora-infra/github2fedmsg",
                 "has_issues": True,
-                "master_branch": "develop",
-                "forks": 0,
-                "open_issues_count": 1,
-                "watchers": 2,
-                "name": "github2fedmsg",
+                "master_branch": "develop",
+                "forks": 0,
+                "open_issues_count": 1,
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/tests/github.py]: Is this changing an existing test? If so, I don't think we should, add new one, deprecate the old one is fine, but we shouldn't change existing ones, imho.

**Comment by pypingou** [comment]
The change to the processor looks fine but the changes to the tests are a little odd, it looks like some of the existing tests have been changed/edited. Is that the case?
We shouldn't change them since we need to maintain backward compatibility and the old messages are still present in datagrepper.

File: fedmsg_meta_fedora_infrastructure/tests/github.py (hunk 1 above)
Comment (sayanchowdhury) [inline fedmsg_meta_fedora_infrastructure/tests/github.py]: Nope, I did not change any of the existing test-cases. Just added the new ones.

**Comment by sayanchowdhury** [comment]
None of existing tests were altered. Though, I'll just go through the change once to make sure nothing has been removed.

**Comment by sayanchowdhury** [comment]
Moved the tests to the end of the file. Looks more sane now.

**Comment by pypingou** [comment]
Thanks, it's easier to read indeed, 👍 for me!

**Comment by sayanchowdhury** [comment]
Thanks :+1:

**Pull request merged**